agent/agent.go

```diff
@@ -273,8 +312,15 @@ func (r *Agent) setupSink(ctx context.Context, sr recipe.PluginRecipe, stream *s
 			retryNotification,
 		)
 
-		success := err == nil
-		r.monitor.RecordPlugin(recipe.Name, sr.Name, "sink", success)
+		pluginInfo.Success = err == nil
+		for _, val := range stream.subscribers {
+			pluginInfo.BatchSize += val.batchSize
+		}
```

Comment:
Why would the batch size be determined by the number of subscribers?

Reply:
any suggestion on how to calculate the batch size for sinks?

Reply:
cc: @spy16 @haveiss